omod/pom.xml

```diff
@@ -76,6 +76,13 @@
             <artifactId>uiframework-api</artifactId>
         </dependency>
 
+		<!-- version comperison -->
+		<dependency>
+    		<groupId>org.apache.maven</groupId>
+    		<artifactId>maven-artifact</artifactId>
+    		<version>3.6.3</version>
```

Comment:
you dont need to add a new dependence just to compare a string or an integer .
you can use a lot of java utils or Openmrs utils to compare versions